test/clj_http/test/client_test.clj

```diff
@@ -1539,6 +1539,8 @@
 (deftest t-coercion-methods
   (let [json-body (ByteArrayInputStream. (.getBytes "{\"foo\":\"bar\"}"))
         json-ms949-body (ByteArrayInputStream. (.getBytes "{\"foo\":\"안뇽\"}" "MS949"))
+        xml-body (ByteArrayInputStream. (.getBytes "<?xml version=\"1.0\" encoding=\"UTF-8\" ?><foo>bar</foo>"))
+        xml-auto-body (ByteArrayInputStream. (.getBytes "<?xml version=\"1.0\" encoding=\"UTF-8\" ?><foo>bar</foo>"))
```

Comment:
I had to make these two separate things because it would progress the stream in the first assert, then it would be EOF immediately for the second assert. Not sure if there is a cleaner way off the top of my head; but that is why it is here twice